webrender_traits/src/types.rs

```diff
@@ -372,6 +372,35 @@ impl Into<f64> for SubpixelOffset {
 }
 
 #[derive(Clone, Hash, PartialEq, Eq, Debug, Deserialize, Serialize, Ord, PartialOrd)]
+pub struct SubpixelPoint {
```

Comment:
I tried to fix this, but had to implement PartialOrd for Point2D<SubpixelOffset>. When I tried to do that, I can't because you aren't allowed to implement traits for types outside your crate.